opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -291,23 +291,25 @@ def get_aggregated_resources(
     :param timeout: Number of seconds to wait for each detector to return
     :return:
     """
-    final_resource = initial_resource or _EMPTY_RESOURCE
-    detectors = [OTELResourceDetector()] + detectors
+    detectors_merged_resource = initial_resource or _EMPTY_RESOURCE
```

Comment:
To be clear, I'm suggesting this change
```suggestion
    detectors_merged_resource = initial_resource or Resource.create()
```

then L315 would jsut be 

```py
return detectors_merged_resource
```

Reply:
Sure I like this solution 😄 It makes the default behavior add those important attributes and users can go out of their way to remove them if they want! I just hope for the sake of the business metrics people who count on these attributes being set that the users won't remove them 🙂